examples/des/blendset.yaml

```diff
@@ -0,0 +1,205 @@
+    type: DES_PSFEx
+
+    # The image_pos doesn't matter much for the MEDS output, but it is used to get the
+    # right PSF from the PSFEx file.
+    image_pos :
+        type : XY
+        x : { type : Random, min: 1, max: 2048 }
+        y : { type : Random, min: 1, max: 4096 }
+
+
+gal:
+    # Choose random galaxies from the COSMOS catalog.
+    type: COSMOSGalaxy
+    index: { type: Random }
+
+    # Either of these will work here, but the real galaxies are much slower on such a small
+    # simulation, since they require a lot of setup time to read in each of the RealGalaxy
+    # image files.  On a typically large simulation, this time gets amortized over a lot
+    # of objects, so the difference isn't as huge.  So to keep the running time reasonable,
+    # we use parameteric here.
+    gal_type: parametric
+    #gal_type: real
+
+    # Randomly spin the original profile
+    rotate: { type: Random }
+
+    # The COSMOS galaxy sample is shallower than we want, so make them a bit smaller to simulate
+    # a fainter sample.  Don't worry about the flux, since we'll just adjust the noise to
+    # an appropriate level.
+    dilate: 0.6
+
+    # This works out about right for the sky level we chose below.  The galaxies mostly seem
+    # to have pretty decent S/N based on visual inspection of image.
+    scale_flux: 1000
+
+    # Just a constant shear for now.
+    shear:
+        type: G1G2
+        g1: 0.03
+        g2: -0.02
+
+stamp:
+    # In blend.py (loaded by the modules field above), we define a custom stamp type
+    # called Blend.
+    # It has one galaxy centered (more or less -- cf image.offset) on the stamp, and then
+    # some number of neighbors also appearing on the stamp placed randomly within some range
+    # of allowed separations.
+    type: BlendSet
+
+    # Up to 5 neighbors.  Each neighbor has a prob of p=0.3 of being included.
+    n_neighbors:
+        type: RandomBinomial
+        N: 5
+        p: 0.3
+
+    # The neighbors cannot be too close (assume they are rejected as unusable) or too far
+    # (not really a blend).
+    min_sep: 2   # arcsec -- clearly overlapping.  Too hard to deblend.
+    max_sep: 10  # arcsec -- this would be off the edge of the stamp.  Not really blended.
+
+    # Offset is just random within central pixel.
+    offset:
+        type: XY
+        x: { type: Random, min: -0.5, max: 0.5 }
+        y: { type: Random, min: -0.5, max: 0.5 }
+
+
+image:
+    pixel_scale : 0.26
+
+    random_seed: 8675309
+
+    # Poisson noise
+    noise:
+        type: Poisson
+        sky_level: 800
+        whiten: True  # Only relevant for gal_type = real, but it doesn't hurt to leave this in.
+
+
+input:
+    cosmos_catalog :
+        # Use the same COSMOS catalog choices we made in meds.yaml
+        template : "meds.yaml:input.cosmos_catalog"
+
+        # Also impose a minimum size, which we didn't there, to avoid very small galaxies.
+        min_hlr : 1.0  # arcsec
```

Comment:
Really? Won't that chuck out nearly everything?  (I'm looking at fig 12 in the GREAT3 handbook)  Also, that isn't particularly small.